packages/react/tsconfig.build.json

```diff
@@ -15,6 +15,7 @@
     "src/utils/story-helpers.tsx",
     "**/*.stories.tsx",
     "**/*.test.tsx",
+    "**/*.figma.tsx",
```

Comment:
@joshblack neither this now the change in `packages/react/tsconfig.json` seems to make typescript ignore my files. Any idea why?

Reply:
@lukasoppermann what are you looking for it to be ignored from? Pulled it down real quick and it seemed to be ignored from the build but just wanted to check-in to make sure I knew what you were looking to do 👀

Reply:
I want it to be ignored from the type checking as well. This would allow me to remove this line: https://github.com/primer/react/pull/4545/files#diff-5fe13549f218c483abf30bee0a4a3001cd58d382586bfedf07299cdbe7721961R2